src/content/docs/guides/migrate-eslint-prettier.mdx

```diff
@@ -0,0 +1,187 @@
+The source of a rule can be found on the page describing the rule.
+You can also find the equivalent Biome rule from an ESLint rule using the [dedicated page](/linter/rules-sources).
+
+To ease the migration, Biome provides the `biome migrate eslint` subcommand.
+This subcommand will read your ESLint configuration and attempt to port its settings to Biome.
+The subcommand is able to handle both the legacy and the flat configuration files.
+It supports the `extends` field of the legacy configuration and loads both shared and plugin configurations.
+The subcommand also migrates `.eslintignore`.
+
+Given the following ESLint configuration:
+
+```json title=".eslintrc.json"
+{
+  "extends": ["plugin:unicorn/recommended"],
+  "plugins": ["unicorn"],
+  "ignore_patterns": ["dist/**"],
+  "globals": {
+    "Global1": "readonly"
+  },
+  "rules": {
+    "eqeqeq": "error"
+  },
+  "overrides": [
+    {
+      "files": ["tests/**"],
+      "rules": {
+        "eqeqeq": "off"
+      }
+    }
+  ]
+}
+```
+
+And the following Biome configuration:
+
+```json title="biome.json"
+{
+	"linter": {
+		"enabled": true,
+		"rules": {
+			"recommended": true
+		}
+	}
+}
+```
+
+Run the following command to migrate your ESLint configuration to Biome.
+
+<PackageManagerBiomeCommand command="migrate eslint --write"/>
+
+The subcommand overwrites your initial Biome configuration.
+For example, it disables `recommended`.
+This results in the following Biome configuration:
+
+```json title="biome.json"
+{
+	"organizeImports": { "enabled": true },
+	"linter": {
+		"enabled": true,
+		"rules": {
+			"recommended": false,
+			"complexity": {
+				"noForEach": "error",
+				"noStaticOnlyClass": "error",
+				"noUselessSwitchCase": "error",
+				"useFlatMap": "error"
+			},
+			"style": {
+				"noNegationElse": "off",
+				"useForOf": "error",
+				"useNodejsImportProtocol": "error",
+				"useNumberNamespace": "error"
+			},
+			"suspicious": {
+				"noDoubleEquals": "error",
+				"noThenProperty": "error",
+				"useIsArray": "error"
+			}
+		}
+	},
+	"javascript": { "globals": ["Global1"] },
+	"overrides": [
+		{
+			"include": ["tests/**"],
+			"linter": { "rules": { "suspicious": { "noDoubleEquals": "off" } } }
+		}
+	]
+}
+```
+
+The subcommand needs Node.js to load and resolve all the plugins and `extends` configured in the ESLint configuration file.
+For now, `biome migrate eslint` doesn't support configuration written in YAML.
+
+By default, Biome doesn't migrate inspired rules.
+You can use the CLI flag `--include-inspired` to migrate them.
+
+<PackageManagerBiomeCommand command="migrate eslint --write --include-inspired"/>
+
+Note that you are unlikely to get exactly the same behavior as ESLint because Biome has chosen not to implement some rule options or to deviate slightly from the original implementation.
+
+Since ESLint takes VCS ignore files into account, we recommend that you enable Biome's [VCS integration](/guides/integrate-in-vcs).
+
```

Comment:
At the end, we might want to add a `:::caution` and tell users that an eslint configuration that contains extraneous plugins (plugins that Biome doesn't know how to handle) might cause an error

Reply:
In my knowledge this doesn't cause any error?

Reply:
I encountered an error that I reported on Discord